docs/product-user-guide/src/main/asciidoc/data-sets-caching-con.adoc

```diff
@@ -1,27 +1,28 @@
 [id='data_sets_caching_con']
 = Caching
 
```

Comment:
This section lacks a lot of the content available in the community documentation. Notice, caching settings are pretty advanced and require proper understanding. Can you please expand with more content? https://docs.jboss.org/jbpm/release/7.7.0.Final/jbpm-docs/html_single/#_wb.datasetcachesettings

Reply:
This suggestion can be taken up post GA release as most of the community content available is conceptual. Will need to figure out how much community content can be included here.